config/example/config.txt

```diff
@@ -1,21 +1,21 @@
-## Server name: This appears at the top of the screen in-game. In this case it will read "spacestation13: station_name" where station_name is the randomly generated name of the station for the round. Remove the # infront of SERVERNAME and replace 'spacestation13' with the name of your choice
-# SERVERNAME spacestation13
+## Server name: This identifies your server in the server list. It may also be used as the title of the game window.
+SERVERNAME ParaCode Test
 
 ## Server tagline: This appears on the hub entry.
-#SERVER_TAG_LINE The Perfect Mix of RP & Action
+SERVER_TAG_LINE ParaCode Test
 
 ## Server extra features: This appears in the feature list on the hub entry.
-#SERVER_EXTRA_FEATURES medium RP, varied species/jobs/modes
+SERVER_EXTRA_FEATURES Medium RP, varied species/jobs.
+
+## Download the RSC from the web - this needs to be regenerated every time the code is changed.
+#RESOURCE_URLS http://www.paradisestation.org/windows/paradise.rsc.zip
```

Comment:
I would put this in, since this is for test servers and their branch RSCs may be **majorly** different than the production RSC

Reply:
Did you mean... that you would leave this blank? Also the reason its there is so that anyone usinjg our codebase can become familiar with how to use the option.

Reply:
Yeah I meant `wouldn't`. Not used to this laptop's keyboard. If its commented out that should be fine though.